scripts/ducttape_nolint_protoc.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+grep -Rn 'package \(\w\+\)$' types/*.pb.go | cut -d':' -f1 | while read F;do backup="$F.bak";sed "s/package \([a-zA-Z]\{1,\}\)$/\/\/nolint: gas<NEWLINE_HEREXXXX>&/1" "$F" | awk -F"<NEWLINE_HEREXXXX>" '{ if (NF >= 2) { print $1"\n"$2 } else { print $1 } }' > "$backup";mv "$backup" "$F";done
```

Comment:
too complicated... maybe we should just do something like `sed  "/package types/ { N; s/\/\/nolint: gas\n/package types\n&/ }" types/types.pb.go`

Reply:
I will do later